core/src/main/java/com/onelogin/saml2/http/HttpRequest.java

```diff
@@ -27,21 +37,45 @@
      * @throws NullPointerException if requestURL is null
      */
     public HttpRequest(String requestURL) {
```

Comment:
Maybe we should deprecate this constructor, pointing to the other one, and mentioning that not providing the queryString can cause issues with signature verification when using the HTTP Redirect binding? @pitbulk?